src/js/components/ConfirmationModal.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+const ModalConfirm = (props) => {
+  return (
+    <div className={'confirmation-blurred-blocker'+ (props.showing ? ' showing-blurred-blocker' : '')}>
+      <div className="confirmation-modal">
+      <div>
+        <h4>Are you sure?</h4>
```

Comment:
Agreed, I'll key into the same logic you're using for the institutions page warning